apps/www/components/LaunchWeek/12/LWMeetups.tsx

```diff
@@ -0,0 +1,152 @@
+        <p className="text-base xl:max-w-md mb-2">
+          Join our live community-driven meetups to Launch Week 12 with the community, listen to
+          tech talks and grab some swag.
+        </p>
+      </div>
+      <div className="col-span-1 xl:col-span-7 xl:col-start-6 w-full max-w-4xl flex flex-wrap gap-x-3 gap-y-1">
+        {meets &&
+          meets
+            ?.sort((a, b) => (new Date(a.date) > new Date(b.date) ? 1 : -1))
+            .map((meetup: Meetup, i: number) => {
+              const startAt = new Date(meetup.date)
+              const endAt = addHours(new Date(meetup.date), 3)
+              const after = now > startAt
+              const before3H = now < endAt
+              const liveNow = after && before3H
+
+              return (
+                <>
+                  <button
+                    key={meetup.id}
+                    onClick={() => handleSelectMeetup(meetup)}
+                    onMouseDown={() => handleSelectMeetup(meetup)}
+                    title={liveNow ? 'Live now' : undefined}
+                    className={cn(
+                      'h-10 group inline-flex md:hidden items-center flex-wrap text-3xl',
+                      'text-foreground-muted hover:!text-foreground !leading-none transition-colors',
+                      meetup.id === activeMeetup?.id && '!text-foreground',
+                      liveNow && 'text-foreground-light'
+                    )}
+                  >
+                    {liveNow && (
+                      <div className="w-2 h-2 rounded-full bg-brand mr-2 mb-4 animate-pulse" />
+                    )}
+                    <span>{meetup.location}</span>
+                    {i !== meets.length - 1 && ', '}
+                  </button>
+                  <Link
+                    key={`meetup-link-${meetup.id}`}
+                    href={meetup.link ?? ''}
+                    target="_blank"
+                    onClick={() => handleSelectMeetup(meetup)}
+                    onMouseOver={() => handleSelectMeetup(meetup)}
+                    title={liveNow ? 'Live now' : undefined}
+                    className={cn(
+                      'hidden h-10 group md:inline-flex items-center flex-wrap text-4xl',
+                      'text-foreground-muted hover:!text-foreground !leading-none transition-colors',
+                      meetup.id === activeMeetup?.id && '!text-foreground',
+                      liveNow && 'text-foreground-light'
+                    )}
+                  >
+                    {liveNow && (
+                      <div className="w-2 h-2 rounded-full bg-brand mr-2 mb-4 animate-pulse" />
+                    )}
+                    <span>{meetup.location}</span>
+                    {i !== meets.length - 1 && ', '}
+                  </Link>
+                </>
+              )
+            })}
+      </div>
+      <Link
+        href={activeMeetup?.link ?? '#'}
+        target="_blank"
```

Comment:
Typically if target is _blank, will also need `rel="noreferrer"` ([ref](https://codedamn.com/news/nextjs/next-js-open-link-in-new-tab-guide))